ray-operator/controllers/ray/raycluster_controller_test.go

```diff
@@ -255,6 +255,38 @@ var _ = Context("Inside the default namespace", func() {
 				listResourceFunc(ctx, &workerPods, filterLabels, &client.ListOptions{Namespace: "default"}),
 				time.Second*15, time.Millisecond*500).Should(Equal(1), fmt.Sprintf("workerGroup %v", workerPods.Items))
 		})
+
+		It("should increase replicas past maxReplicas", func() {
+			// increasing replicas to 5, which is greater than maxReplicas (4)
+			err := retryOnOldRevision(DefaultAttempts, DefaultSleepDurationInSeconds, func() error {
+				Eventually(
+					getResourceFunc(ctx, client.ObjectKey{Name: myRayCluster.Name, Namespace: "default"}, myRayCluster),
+					time.Second*9, time.Millisecond*500).Should(BeNil(), "My raycluster = %v", myRayCluster)
```

Comment:
Just out of curiosity, why did we choose `time.Second*9` as the timeout and `time.Millisecond*500` as the polling interval?

Reply:
It's copy-pasted from a previous block of logic in the same test.

Reply:
Got it. Thank you!